cli/aeon/cmd/connect.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+type Ssl struct {
```

Comment:
Missed comment.

Reply:
And dots in the end of the commentaries.